pennylane/ops/op_math/controlled_ops.py

```diff
@@ -141,8 +146,8 @@ def __init__(
         self._name = "ControlledQubitUnitary"
 
     def _controlled(self, wire):
-        ctrl_wires = self.control_wires + wire
-        values = None if self.control_values is None else self.control_values + [True]
+        ctrl_wires = wire + self.control_wires
+        values = None if self.control_values is None else [True] + self.control_values
```

Comment:
It wasn't, but now that it inherits from `Controlled`, this conflicts with the convention with all other controlled operations, which caused some test cases to fail, which was how I caught it.

Reply:
I don't think it was, but after the clean-up, some tests were failing because of this. It would be consistent with how all the other nested controlled operations order their control wires.